wall.h

```diff
@@ -0,0 +1,8 @@
+#ifndef WALL_H
+#define WALL_H
+
+void wall(const char *message, ...);
+void wall_ssh_users(const char *message);
+void wall_welcome(void);
```

Comment:
public functions and define guards should be prefixed with `pv_` or `PV_`  as appropriate